app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    """ That will move a file from one location to another """
+    # Ex: mv file.txt first_dir/second_dir/third_dir/file2.txt
+
+    (cmd,
+     file_in,
+     file_out) = (
+        command.split(" ")[0],
+        command.split(" ")[1],
+        command.split(" ")[2])
```

Comment:
Do not use split three times
```suggestion
    (cmd,
     file_in,
     file_out) = command.split()
     ```